action.rb

```diff
@@ -1,25 +1,35 @@
 class Action
   attr_reader :headers, :body, :request
 
-  def initialize &block
+  def initialize(&block)
     @block = block
-    @status = 200
-    @headers = {'Content-Type' => 'text/html'}
-    @body = ''
+    @headers = {'Content-Type' => 'application/json'}
   end
 
-  def status value = nil
-    value ? @status = value : @status
+  def status(value = nil)
+    value ? @status = value : @status || 200
```

Comment:
@brandondees care to give an example? And i'll do that refactor now. Not sure what you mean.